dll/win32/advapi32/misc/evntrace.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     advapi32.dll Event tracing stubs
+ * COPYRIGHT:   Copyright 2017 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+#include <wmistr.h>
+#include <evntrace.h>
+
+WINE_DEFAULT_DEBUG_CHANNEL(advapi);
+
+
+TRACEHANDLE
+WINAPI
+OpenTraceA(IN PEVENT_TRACE_LOGFILEA Logfile)
+{
+    UNIMPLEMENTED;
```

Comment:
Wine's version also does `SetLastError(ERROR_ACCESS_DENIED);`. Think that might be useful?